tracing/README.md

```diff
@@ -0,0 +1,188 @@
+setup of tracing, and add your favorite tracer implementation to the classpath (currently Zipkin is the only
+favorite tracer implemented, though SPI is available).
+
+Example of pom.xml dependencies:
+```xml
+<dependency>
+    <!-- general support for tracing -->
+    <groupId>io.helidon.microprofile.tracing</groupId>
+    <artifactId>helidon-microprofile-tracing</artifactId>
+</dependency>
+<dependency>
+    <!-- Zipkin tracer implementation -->
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+```
+
+To customize configuration of zipkin, see `ZipkinTracerBuilder` javadoc. Basics:
+```yaml
+tracing:
+  # required
+  service: "service-name"
+  # default is "localhost"
+  host: "zipkin"
+  # default is 9411
+  port: 9411
+```
+
+## Usage in Helidon SE
+To use tracing in SE, add a dependency on `helidon-tracing` and a tracer
+implementation and register tracer with server configuration.
+
+pom.xml:
+```xml
+<dependency>
+    <!-- to add tracer builder -->
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing</artifactId>
+</dependency>
+<dependency>
+    <!-- to add zipkin support -->
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+```
+
+code using config:
+```java
+return ServerConfiguration.builder()
+                .config(config.get("webserver"))
+                .tracer(TracerBuilder.create(config.get("tracing"))
+                                        .buildAndRegister())
+                .build();
+```
+
+To customize configuration of zipkin, see `ZipkinTracerBuilder` javadoc. Basics:
+```yaml
+tracing:
+  # required
+  service: "service-name"
+  # default is "localhost"
+  host: "zipkin"
+```
+
+### Usage of Jersey client outside of Helidon MP
+Jersey client can be used (maybe standalone, or in Helidon SE) and traced.
+Simple add `helidon-tracing-jersey-client` as a dependency and correctly configure
+the request properties. The client expects Tracer to be configured (see zipkin examples above)
+ 
+Note that in Helidon MP, this would work automatically with no additional configuration if you 
+carry out the steps described in "Usage in Helidon MP".
+
+```xml
+<dependency>
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-jersey-client</artifactId>
+</dependency>
+```
+
+And in code:
+```java
+response = webTarget
+            .request()
+            // tracer information - not required if global tracer should be used
+            .property(ClientTracingFilter.TRACER_PROPERTY_NAME, tracer)
+            // the current tracing span context to be used as a parent for outbound request
+            // if not provided a new span with no parent would be created
+            .property(ClientTracingFilter.CURRENT_SPAN_CONTEXT_PROPERTY_NAME, spanContext)
+            .get();
+```
+
+#Modules
```

Comment:
Just curious... What's the rationale for the current listing order of the modules?

helidon-tracing
helidon-tracing-jersey-client
helidon-tracing-jersey
helidon-microprofile-tracing
helidon-tracing-zipkin